src/classes/Commands/sub-classes/Manager.ts

```diff
@@ -405,6 +405,28 @@ export default class ManagerCommands {
         });
     }
 
+    async haltCommand(steamID: SteamID): Promise<void> {
+        if (this.bot.isHalted) {
+            this.bot.sendMessage(steamID, 'Already halted, nothing to halt');
+            return;
+        }
+        this.bot.sendMessage(steamID, '⌛ Halting...');
+        await this.bot.halt();
+    }
+
+    async unhaltCommand(steamID: SteamID): Promise<void> {
+        if (!this.bot.isHalted) {
+            this.bot.sendMessage(steamID, 'Not halted, nothing to unhalt');
+            return;
+        }
+        this.bot.sendMessage(steamID, '⌛ Unhalting...');
+        await this.bot.unhalt();
+    }
```

Comment:
And also for `!halt` command I guess... if x minutes after start/restart has passed, then can use that command.

Reply:
Not sure if we should do it, since it still can be worked around by using `!restart`. Maybe just leave it to user's choice?

If we really do need this cooldown, I think of setting it small, like 5 minutes since restart and 10 minutes between halt/unhalt.

Reply:
Alright, that seems reasonable.